Kickstarter-iOS/ViewModels/AppDelegateViewModel.swift

```diff
@@ -312,12 +272,13 @@ AppDelegateViewModelOutputs {
 
     self.pushTokenSuccessfullyRegistered = self.deviceTokenDataProperty.signal
       .map(deviceToken(fromData:))
+      .on(value: { print("📲 [Push Registration] Push token generated: (\($0))") })
```

Comment:
I've added this as a side-effect so this will print regardless of the call to the back-end success/failure